content/docs/2.5/scalers/newrelic.md

```diff
@@ -0,0 +1,70 @@
++++
+title = "New Relic"
+availability = "2.5+"
+maintainer = "Community"
+description = "Scale applications based on NewRelic NRQL"
+layout = "scaler"
+go_file = "newrelic_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `New Relic` trigger that scales based on a New Relic metric.
+
+```yaml
+triggers:
+  - type: new-relic
+    metadata:
+      # Required: Account - Subaccount to run the query on
+      Account: 1234567
+      # Required: QueryKey - Api key to connect to New Relic
+      QueryKey: "NRAK-xxxxxxxxxxxxxxxxxxxxxxxxxxx"
+      # Optional: nrRegion - Region to query data for
+      Region: "US"
+      # Required: metricName
+      metricName: "duration"
+      # Required: nrql
+      nrql: "SELECT average(duration) from Transaction where appName='SITE' TIMESERIES"
+      # Required: threshold
+      threshold: 100
+      # Optional: nrLogLevel
+      LogLevel: "info"
+```
+
+**Parameter list:**
+
+- `Account` - The account within New Relic that the request should be targeted against.
+- `QueryKey` - The API key that will be leveraged to connect to New Relic and make requests. [official documentation](https://docs.newrelic.com/docs/apis/intro-apis/new-relic-api-keys/)
+- `Region` - The region to connect to for the New Relic apis. (Values: `LOCAL`, `EU`, `STAGING`, `US`, Default: `US`, Optional)
+- `metricName` - The metric to pull from the query result.
+- `nrql` - The New Relic query that will be run to get the data requested. [official documentation](https://docs.newrelic.com/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/)
+- `threshold` - A threshold that is used as the `targetAverageValue` in the HPA configuration.
+- `LogLevel` - The logging level of the underlying New Relic class to aid in debugging. (Values: `debug`, `error`, `fatal`, `info`, `trace`, `warn`, Default: `info`, Optional)
+
+### Authentication Parameters
+
```

Comment:
```suggestion

As of today, using `TriggerAuthentication` is not supported.

```

Reply:
Updated to include this statement, it should change relatively quickly based upon the requested change for the key auth which authentication params would be valid.

Reply:
Re-opening - Can you please align with the other scalers so that it's more consistent please?

For example https://keda.sh/docs/2.4/scalers/metrics-api/#authentication-parameters

Reply:
Re-opening